docs/Duplicate-Method-Call.md

```diff
@@ -42,7 +42,56 @@ The approach you take will depend on balancing other factors in your code.
 
 ## Current support in Reek
 
-Reek's Duplicate Method Call detector checks for repeated identical method calls within any one method definition. This is intended to complement the checks performed by tools such as [Flay](http://ruby.sadi.st/Flay.html) and [Simian](http://www.redhillconsulting.com.au/products/simian/).
+Reek's Duplicate Method Call detector checks for repeated identical method calls within
+any one method definition. This is intended to complement the checks performed by tools
+such as [Flay](http://ruby.sadi.st/Flay.html) and [Simian](http://www.redhillconsulting.com.au/products/simian/).
+
+Unfortunately there are some edge cases Reek can't reliably detect due to Ruby's dynamic nature.
+
+Consider this code:
+
+```Ruby
+class Foo
+  def bar(switch)
+    case switch
+    when :a
+      ->(arg) { arg.call_me(:maybe); do_something }
+    when :b
+      ->(arg) { arg.call_me(:maybe); do_something_else }
+    when :c
+      ->(arg) { arg.call_me(:maybe); do_something_different }
+    end
+  end
+end
+```
+
+In regards to duplicate method calls this should be fine since this happens inside a lambda expression.
+However as mentioned above Reek can't reliably detect such cases and will report:
```

Comment:
I wouldn't say that the code is fine. Maybe change it to something like: 'Reek cannot reliably detect that each call's receiver is a different `arg`, and will report:'